.github/workflows/ci.yml

```diff
@@ -84,8 +84,12 @@ jobs:
         run: yarn lerna run test-unit --scope=@sequelize/utils
       - name: Unit tests (core - mariadb)
         run: yarn lerna run test-unit-mariadb --scope=@sequelize/core
+      - name: Unit tests (mariadb package)
+        run: yarn lerna run test-unit --scope=@sequelize/mariadb
       - name: Unit tests (core - mysql)
         run: yarn lerna run test-unit-mysql --scope=@sequelize/core
+      - name: Unit tests (mysql package)
+        run: yarn lerna run test-unit --scope=@sequelize/mysql
```

Comment:
I don't see a change in the package.json of mysql. Shouldn't this fail?

Reply:
Added